vite.config.ts

```diff
@@ -22,6 +22,15 @@ export default defineConfig({
       },
     }),
   ],
+  server: {
```

Comment:
This was required to make Firebase auth work in the dev environment. Will it work in production? Find out on the next episode of Deploying to Prod.